os/arch/xtensa/src/esp32/esp32_timerisr.c

```diff
@@ -125,8 +125,8 @@ int64_t get_instant_time(void)
 	compare = xtensa_getcompare();
 	diff = xtensa_getcount() - compare;
 	int ticks = clock_systimer();
-    int64_t ms = TICK2MSEC(ticks) + (diff*1000) / g_tick_divisor;
-    return ms;
+	int64_t ms = TICK2MSEC(ticks) + (diff*1000) / g_tick_divisor;
```

Comment:
Actually you should not include two different kinds of change in a commit.
Could you split coding rules changes from this commit and fix one more violation?
```
(diff * 1000)
```
needs spaces around `*`

Reply:
done.